en/docs/design/api-policies/overview.md

```diff
@@ -6,8 +6,6 @@ Policy is dependent on the API Gateway that you have configured with WSO2 API Ma
 
 - **Regular Gateway (Synapse Gateway)**
 
-- **Choreo Connect**
-
```

Comment:
Shall we remove the below line as we no longer have to list the gateway types?

```
Policy is dependent on the API Gateway that you have configured with WSO2 API Manager. The API Policy feature supports the following two Gateway types:
```

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7875/commits/d065b8164a5b1f79e3df2a35d72ee6d8fcdbffba